projects/farmworker-wallet-os.md

```diff
@@ -0,0 +1,120 @@
+
+ 
+The first digital service launched supported the UFW Foundation’s emergency relief efforts during the height of the COVID-19 pandemic. The solution was used to plan, manage and operate over 500 in-person community events where over $15 million in resources were distributed to over 40K families. Additionally, it has allowed us to better understand the unique challenges of building for underserved communities and why interoperable, secure, and privacy preserving technologies are key to addressing these challenges.  
+
+  
+You can read more about our journey on [our blog](https://www.entidad.io/blog). 
+
+# Alignment with the OpenWallet Foundation Mission
+The **Farmworker Wallet OS** project is a community of contribution led by Entidad and the United Farm Workers Foundation (UFWF) with the goal of furthering the adoption of an open, secure, interoperable digital wallet solution that makes it easier for farm worker communities to access life-changing social and human services. The open-source software components maintained by this community will enable non-profit and social good organizations to build wallet-driven solutions on top of the Mendix low-code platform. Collaboration with app makers from the broader Mendix developer community will help inform and guide the Farmworker Wallet OS project with industry proven best practices, leading to the implementation of reusable functionality that is aligned with the goals of the Open Wallet Foundation (to preserve user choice, security and privacy). The community will also inspire the next generation of low-code app builders to learn and expand their skill sets with decentralized (web3) technologies. 
+
+# Current Code of Conduct
+[Farmworker Wallet OS Community Code of Conduct](https://github.com/Entidad/.github/blob/main/CODE_OF_CONDUCT.md)
+
+# TAC Sponsor
+_Include the name of a sponsor from the TAC, if identified (a sponsor helps mentor projects)_
+
+# Project License
+Because of the social impact origins of the Farmworker Wallet OS project, it is important to the initial contributors that source code not be used in ways that could bring harm to underserved communities.  As such, we propose to use the [Hippocratic License](https://firstdonoharm.dev/), Version 3.0, October 2021.  This license was created by the [Organization for Ethical Source](https://ethicalsource.dev/) in partnership with [Corporate Accountability Lab](https://corpaccountabilitylab.org/). The Hippocratic License is largely based on the Apache 2.0 license, with the addition of ethical standards. [Farmworker Wallet OS project proposed license](https://firstdonoharm.dev/version/3/0/eco-extr-law-media-mil-soc-sv-usta.html).
```

Comment:
Based on the [OWF Project Lifecycle](https://openwallet-foundation.github.io/tac/governance/project-lifecycle/), we would need to get approval of this license from the Governing Board (with input from LF legal).

> Document an intellectual property policy that leverages the Apache 2.0 license or an open license that has been approved by the OpenWallet Foundation's Governing Board.

Reply:
After checking with The Linux Foundation, we cannot host a project using this license. Among other restrictions, it prohibits the use of software licensed under this license by any government that maintains a military, which would disqualify this software for use by most European and other governments.

To provide additional detail, The Linux Foundation open source projects must use licenses approved as "open" by the [Open Source Initiative](https://opensource.org/) ("OSI").  OSI maintains a list of licenses approved as open, that meet its definition of "open".  The OSI definition of "open" can be reviewed here: [https://opensource.org/osd/](https://opensource.org/osd/).  In particular, this Hippocratic license has not been approved as open by OSI, and it would appear to conflict with at least two requirements of the OSI definition:

> 5. No Discrimination Against Persons or Groups
> The license must not discriminate against any person or group of persons.
> 
> 6. No Discrimination Against Fields of Endeavor
> The license must not restrict anyone from making use of the program in a specific field of endeavor. For example, it may not restrict the program from being used in a business, or from being used for genetic research.

This Hippocratic license provides extensive restrictions on persons, groups and fields of endeavor.

Reply:
Thank you for checking with the Linux Foundation.  We understand the position from both the Linux and OpenWallet Foundations.  We will replace the Hippocratic license from our proposal with the recommended Apache 2.0 license.